### DISCUSSION TITLE
youtube-dl doesn't update beyond 2021.12.17

### DISCUSSION THREAD

**Discussion by Rommsel** [MiczFlor/RPi-Jukebox-RFID#2011] [Q&A]
Hi,
the youtube download doesn't work for me. When I put in the test line:
`youtube-dl -f bestaudio --extract-audio --audio-format mp3 https://www.youtube.com/watch?v=eW4FJI4ZlRo`

I get the message:
"[youtube] eW4FJI4ZlRo: Downloading webpage
**ERROR:** **Unable to extract uploader id**; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output."

I then download the newest youtube-dl with:
`sudo youtube-dl -U`

And it shows:
youtube-dl is up-to-date (2021.12.17)

But I assume that is not up to date and that there should be a version around from 2023 at least. Any idea how to fix this? Thanks for your replies

**Comment by s-martin** [comment]
This seems to be the issue although I'm not yet sure how to fix this:
https://github.com/ytdl-org/youtube-dl/issues/31530
